### DISCUSSION TITLE
Help translating 7th heaven (tiếng việt)

### DISCUSSION THREAD

**Discussion by mitztastic** [tsunamods-codes/7th-Heaven#152] [Q&A] [answered]
I am willing to get a vietnamese localization started for 7H if anyone can assist me with what xml etc files need to be edited.
Regards (M.)

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hello @mitztastic thanks for the offer. I think it will be easier to coordinate on Discord. Feel free to join our Discord channel that you can find here: https://7thheaven.rocks/#download7h Cheers!
